### ISSUE TITLE
Production of Word output tables from simulation summaries

### ISSUE THREAD

**Issue by dimitriskontosBAY** [Roche/crmPack#608]
**Summary**

Production of Word output tables from simulation summaries

**Comment by danielinteractive** [comment]
@0liver0815 @gunhanb is this something interesting in principle for you? Definitely for the backlog but wondering if you have some prototype or need for this.

**Comment by Puzzled-Face** [comment]
@danielinteractive Yes.  Definitely of interest.  In fact, I have much of the necessary infrastructure already in place.  I saw a common need with the [rbqmR](https://github.com/openpharma/rbqmR) project and spun the necessary work out into a standalone package [here](https://github.com/openpharma/autoquarto).  It's based on Quarto, so it should support Word, PDF, HTML etc out-of-the-box.  All that's necessary is creating the necessary `crmPack`-specific templates.
